### PR TITLE
Add a test for rstudio/shiny#3740

### PR DESCRIPTION
**Pull request by cpsievert**
https://github.com/rstudio/shiny/pull/3740

### CODE REVIEW

File: inst/apps/220-execute-hidden-plot/app.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  ")
+)
+
+server <- function(input, output, session) {
+
+  shinyjster::shinyjster_server(input, output)
+
+  has_run <- reactiveVal(FALSE)
+
+  output$plot <- renderPlot({
```
Comment (schloerke) [inline inst/apps/220-execute-hidden-plot/app.R:42]: The error occurs as a "pre" step for `renderPlot()`? So if it makes it to `has_run(TRUE)`, then it is a success?

Comment (cpsievert) [reply inst/apps/220-execute-hidden-plot/app.R:42]: Yep, that's right

Comment (schloerke) [inline inst/apps/220-execute-hidden-plot/app.R:42]:
```suggestion
  # The test in this app before the expression of `renderPlot()` is executed
  # https://github.com/rstudio/shiny/pull/3740
  output$plot <- renderPlot({
```

**Review by schloerke — APPROVED** [review]
LGTM

Maybe should wait to merge once https://github.com/rstudio/shiny/pull/3740 is merged

**Review by jcheng5 — APPROVED** [review]

**Pull request merged**